src/Device/DeviceSelector/BasicDeviceInfo.jsx

```diff
@@ -72,6 +101,8 @@ const BasicDeviceInfo = ({ device, whiteBackground, additionalToggle }) => (
 );
 BasicDeviceInfo.propTypes = {
     device: deviceShape.isRequired,
+    // eslint-disable-next-line react/require-default-props
```

Comment:
If this rule need to be disabled, shall we add it in eslint config?

Reply:
Yes, I will prepare a PR for it.